README.md

```diff
@@ -44,7 +44,7 @@ add it to your CI manually:
 jobs:
   sigstore-python:
     steps:
-      - uses: trailofbits/gh-action-sigstore-python@v0.0.1
+      - uses: trailofbits/gh-action-sigstore-python@v1.0.0
```

Comment:
What is this for?

Reply:
Whoops, that was a fatfinger on my part -- I got the repo mixed up with the GHA for `pip-audit`. Feel free to remove, otherwise I'll take care of it tomorrow.

Reply:
[40ac25a](https://github.com/sigstore/sigstore-python/pull/176/commits/40ac25a3ff272e013afd83ae0b1a0bb7013d1377)